platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/Menu.kt

```diff
@@ -236,11 +238,13 @@ public fun MenuContent(
     Box(
         modifier =
             modifier
-                .shadow(
-                    elevation = style.metrics.shadowSize,
-                    shape = menuShape,
-                    ambientColor = colors.shadow,
-                    spotColor = colors.shadow,
+                .dropShadow(
+                    RoundedCornerShape(style.metrics.cornerSize),
+                    Shadow(
+                        radius = style.metrics.shadowSize,
+                        color = colors.shadow,
+                        offset = DpOffset(0.dp, style.metrics.shadowSize / 2),
```

Comment:
On Swing, it was using the same size from the shadow as the offset, to prevent shadow on top. 

On this Compose call, only half of the value is needed to reproduce the desired UI.

| full size | base-line | half size |
| --- | --- | --- |
| <img width="360" height="144" alt="Image" src="https://github.com/user-attachments/assets/f7a6a3dd-404b-47f3-b2a1-837b843ba571" /> | <img width="360" height="140" alt="Image" src="https://github.com/user-attachments/assets/f5b3d3aa-8985-4d95-bdac-178456679b5b" /> | <img width="360" height="140" alt="Image" src="https://github.com/user-attachments/assets/b8439629-e2c4-4ba5-aa19-89f1fef536dc" />